open_feature/exception/exceptions.py

```diff
@@ -85,3 +85,37 @@ def __init__(self, error_message: str = None):
         @return: the generic TypeMismatchError exception
         """
         super().__init__(error_message, ErrorCode.TYPE_MISMATCH)
+
+
+class TargetingKeyMissingError(OpenFeatureError):
+    """
+    This exception should be raised when the provider requires a targeting key
+    but one was not provided in the evaluation context.
+    """
+
+    def __init__(self, error_message: str = None):
+        """
+        Constructor for the TargetingKeyMissingError. The error code for this type of
+        exception is ErrorCode.TARGETING_KEY_MISSING.
+        @param error_message: a string message representing why the error has been
+        raised
+        @return: the generic TargetingKeyMissingError exception
```

Comment:
I've just noticed that this is a general issue with all the exceptions in this module... but technically this method doesn't actually return anything, would you mind removing the returns on the new exceptions you've added and I'll get the rest cleaned up.

Reply:
Yep sounds good